### DISCUSSION TITLE
Is there a command to upgrade pwsh directly on it?

### DISCUSSION THREAD

**Discussion by GreepTheSheep** [PowerShell/PowerShell#16255] [Q&A]
Hi, I saw that a version is available but I'm asking if there's a possibility to download and install the update directly with a command on it? Would be awesome!

**Comment by LarsKemmann** [comment]
Yes, or a way to opt-in to automatic updates (would be even better 😀).

**Reply by iSazonov** [reply]
See https://github.com/PowerShell/PowerShell/discussions/15510

**Reply by LarsKemmann** [reply]
Okay, that's exciting news! Someone should pin that.

**Reply by iSazonov** [reply]
I see users report `winget upgrade pwsh` works too.

**Comment by rhubarb-geek-nz** [comment]
Both .NET Core and Visual Studio Code are in Linux compatible package repositories, eg https://packages.microsoft.com.
Are the Linux packages for powershell also there? For x86_64, armv7, aarch64 ?
